app/Http/Controllers/MovieController.php

```diff
@@ -2,9 +2,72 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\Celebrity;
+use App\Models\Movie;
 use Illuminate\Http\Request;
 
 class MovieController extends Controller
 {
-    //
+    public function index() //print all data in Moives database
+    {
+
+        return view('movies.index', [
+            'movies' => Movie::orderBy('title', 'ASC')
+            ->orderBy('release_date', 'DESC')
+            ->paginate(25)
+        ]);
+
+    }
+
+    public function show(Movie $movie)
+    {
+
+        $movie->load('celebrities');
```

Comment:
You can do a shortcut and have that in one line ;D 
```
$movies->load(['celebrities', 'genres']);
```

Reply:
Done